src/internal/agent/hooks/pods.go

```diff
@@ -74,6 +79,7 @@ func mutatePod(ctx context.Context, r *v1.AdmissionRequest, cluster *cluster.Clu
 			message.Warnf(lang.AgentErrImageSwap, container.Image)
 			continue // Continue, because we might as well attempt to mutate the other containers for this pod
 		}
+		updatedAnnotations[fmt.Sprintf("zarf.dev/original-init-image-%d", idx)] = container.Image
```

Comment:
Lets make the annotation prefix a constant, maybe even wrap it in a function. Could we also change it from the index to the name of the container?